internal_packages/composer-emoji/lib/emoji-composer-extension.jsx

```diff
@@ -133,16 +134,16 @@ class EmojiComposerExtension extends ComposerExtension {
 
   static applyTransformsToDraft = ({draft}) => {
     const nextDraft = draft.clone();
-    nextDraft.body = nextDraft.body.replace(/<img class="missing-emoji ([a-zA-Z0-9-_]*)" [^<]+>/g, (match, emojiName) =>
-      `<span class="broken-emoji ${emojiName}">${emoji.get(emojiName)}</span>`
+    nextDraft.body = nextDraft.body.replace(/<img class="emoji ([a-zA-Z0-9-_]*)" [^<]+>/g, (match, emojiName) =>
+      `<span class="emoji ${emojiName}">${emoji.get(emojiName)}</span>`
```

Comment:
Is there any way to match the emoji characters themselves with a RegExp? It'd be kind of cool if these functions went from emoji `<img>` => emoji unicode character and back, so that we could handle drafts created in other apps that have unicode character emoji, pasting, etc.